app/src/main/res/layout/notification_dialog.xml

```diff
@@ -25,21 +25,19 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="4dp"
-        android:layout_marginTop="84dp"
         android:ems="10"
+        android:hint="Subtext"
         android:inputType="textPersonName"
-        android:text="Subtext"
```

Comment:
On your two text boxes, I replaced your placeholder text with hints. This creates a better user experience.

Reply:
Good.

Go one step further and externalize the string.  Also, make it something descriptive... more descriptive than Subtext, at least.  :)